site/_includes/docs/_hosted-backend-warning.md

```diff
@@ -0,0 +1,10 @@
+When developing against a FusionAuth Cloud instance with a `.fusionauth.io` address, unless your application also lives at a `.fusionauth.io` address, you'll receive a `403` response code. The hosted backend does not work across origins. This occurs whenever FusionAuth is hosted on a different domain from the application accessing the hosted backend.
+
+To work around this, you can:
+
+* develop locally (`localhost` will work)
+* set up a lightweight proxy to ensure both servers are the same domain
+* stand up a barebones backend with a more liberal cookie policy (https://github.com/FusionAuth/fusionauth-example-react-sdk/tree/main/server[here's an example])
+* set up a [custom domain name for the FusionAuth Cloud instance](/docs/v1/tech/installation-guide/cloud#updating-with-existing-custom-domains) (limited to certain plans).
```

Comment:
It seems that these links are being rendered incorrectly when the file is being included in `.md` vs `.adoc` files:

`.adoc`:
![adoc](https://github.com/FusionAuth/fusionauth-site/assets/1877191/fdba8b4b-a262-48dc-bfab-00d4f2001bdc)

`.md`: 
![md](https://github.com/FusionAuth/fusionauth-site/assets/1877191/479f8ee4-d0c5-40e1-aa2f-bd2f10c4a519)

I couldn't make them work in both files, so maybe we should have 2 `_hosted-backend-warning` files with each extension and different hyperlink syntax.